### DISCUSSION TITLE
What version now? v0.3.0 or v0.3.1?

### DISCUSSION THREAD

**Discussion by Data-san** [CMSgov/price-transparency-guide#209] [Q&A] [answered]
Is the latest schema v03.0 or v03.01? Perhaps they're one and the same. I'm not totally sure per the postings. Ty!

**Comment by shaselton-usds** [comment, marked as answer]
@Data-san 

You should be able to follow the current version here: https://github.com/CMSgov/price-transparency-guide/releases

Also, there is a [version.md](https://github.com/CMSgov/price-transparency-guide/blob/master/VERSION.md) file that is tracking the current version as well.

**Reply by Data-san** [reply]
Thanks much. Any idea how often new/revised schema versions will occur? Right now, it seems to be a monthly cadence, but I assume the schema will stabilize at some point. Is CMS close to that?

Thanks again.

**Reply by shaselton-usds** [reply]
@Data-san Changes will continue as issues are brought up. Agreed that once major issues are address, the schema will stabilize -- there are a couple large buckets of issues that once addressed should slow down the iteration of the schema. 

CMS has planned for a v1.0 3/1/22.

**Reply by Data-san** [reply]
Thanks again. I was on the CMS webinar so aware of the plan.
